### PR TITLE
Adds sniffer plugin

### PR DESCRIPTION
**Pull request by larissa**
Adds the sniffer plugin, a plugin to help enterprises find consumers and suppliers for their products or product categories of interest.

### CODE REVIEW

File: plugins/sniffer/lib/sniffer_plugin.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  def self.plugin_description
+    _("Sniffs product suppliers and consumers near to your enterprise.")
+  end
+
+  def stylesheet?
+    true
+  end
+
+  def js_files
+    ['underscore-min.js', 'sniffer.js'].map{ |j| "javascripts/#{j}" }
```
Comment (brauliobo) [inline plugins/sniffer/lib/sniffer_plugin.rb]: @larissa could you please load these locally? we already have many global javascripts

File: plugins/sniffer/test/integration/sniffer_map_test.rb [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+      )
+    end
+    # Build relationship between products, by defining production inputs:
+    #  ent1   ent2   ent3   ent4
+    # ┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄
+    #   p1◄━┭──p3  ╭─►p5     p7
+    #       │┏━━━━━┵──────╮
+    #       ╰╂─────╮      │
+    #   p2━━━┛ p4  ╰──p6  ╰─►p8
+    #
```
Comment (brauliobo) [inline plugins/sniffer/test/integration/sniffer_map_test.rb]: beautiful, was it @aurium ?

**Comment by brauliobo** [comment]
@larissa only one changed needed, please change it for server update

**Comment by larissa** [comment]
I changed the javascript to load locally. And, yes, it was aurium :D

**Pull request merged**